pkg/cmd/variables/list/list.go

```diff
@@ -54,25 +53,18 @@ func listAllVariables(client *api.Client, out io.Writer, opts *contracts.ListOpt
 	}
 
 	tbl := table.New("ID", "KEY", "VALUE")
-	tbl.WithWriter(out)
-	fields := []string{"GetUuid()", "GetKey()", "GetValue()"}
+	tbl.WithWriter(f.IOStreams.Out)
 
 	if opts.Details {
-		fields = append(fields, "GetSecret()", "GetLastEditor()")
 		tbl = table.New("ID", "KEY", "VALUE", "SECRET", "LAST EDITOR")
 	}
 
 	headerFmt := color.New(color.FgBlue, color.Underline).SprintfFunc()
 	columnFmt := color.New(color.FgGreen).SprintfFunc()
 	tbl.WithHeaderFormatter(headerFmt).WithFirstColumnFormatter(columnFmt)
 
-	rows := printer.BuildRows(resp, fields)
-	for _, row := range rows {
-		if len(row) == 5 {
-			tbl.AddRow(row[0], row[1], row[2], row[3], row[4])
-		} else {
-			tbl.AddRow(row[0], row[1], row[2])
-		}
+	for _, v := range resp {
```

Comment:
I think you need to separate this field, checking if they asked for --details

Reply:
No because it lists based on the columns, as you are already checking above